mycore-base/src/main/java/org/mycore/datamodel/classifications2/MCRCategoryDAOFactory.java

```diff
@@ -44,13 +44,13 @@ public static MCRCategoryDAO getInstance() {
      * @param daoClass new dao class
      */
     public static synchronized void set(Class<? extends MCRCategoryDAO> daoClass) throws ReflectiveOperationException {
-        MCRCategoryDAOHolder.INSTANCE = daoClass.getDeclaredConstructor().newInstance();
+        MCRCategoryDAOHolder.instance = daoClass.getDeclaredConstructor().newInstance();
     }
 
-    // encapsulate the INSTANCE in an inner static class to avoid issues with class loading order
+    // encapsulate the instance in an inner static class to avoid issues with class loading order
     // this is known as "Bill Pugh singleton"
     private static class MCRCategoryDAOHolder {
-        private static MCRCategoryDAO INSTANCE = MCRConfiguration2.getInstanceOfOrThrow(
+        private static MCRCategoryDAO instance = MCRConfiguration2.getInstanceOfOrThrow(
```

Comment:
should be final resp. upper case

Reply:
Can't be final because of: 

```
    public static synchronized void set(Class<? extends MCRCategoryDAO> daoClass) throws ReflectiveOperationException {
        MCRCategoryDAOHolder.instance = daoClass.getDeclaredConstructor().newInstance();
    }
```
Cannot assign a value to final variable 'instance'